src/cfn/template.yaml

```diff
@@ -8,10 +8,11 @@ Globals:
         API_GATEWAY: !Sub https://${RestApi}.execute-api.${AWS::Region}.amazonaws.com/PROD
         COGNITO_IDENTITY_POOL: !Ref CognitoIdentityPool
         COLLECTION_ID: !Ref CollectionId
-        FROM_BUCKET: rekognition-engagement-meter
+        FROM_BUCKET: !Sub solution-builders-${AWS::Region}
         CREATE_CLOUDFRONT_DISTRIBUTION: !Ref CreateCloudFrontDistribution
         REGION: !Ref AWS::Region
         TO_BUCKET: !Ref WebUIBucket
+        VERSION: '0.1'
```

Comment:
Is this version correct? Should it not be something like `1.1`?

Reply:
Not sure, I was thinking of like testing this first and keeping the links to old, and if all works I'll publish as 1.0 with the README updates

Reply:
Makes sense